### DISCUSSION TITLE
Wierd. I get a trojan warning for the zip file.

### DISCUSSION THREAD

**Discussion by Kakail** [openai/whisper#74] [General]
I download the zip file and Windows 10 detect a menace.
I redownloaded and got the same message.

It's about Trojan:Script/Wacatas.H!ml

Anyone else got this?

**Comment by allanlorenz1402** [comment]
[statistics assignment help](https://www.statisticsassignmenthelp.com/) was the reason I scored 90% in my internals during my school in this subject. Statistics always bothered me in my schooldays but, still it was compulsory to pass in all the subjects and the minimum to pass was 50% in internals. And, I was sure that I won't get it on my own. If you feel same then visit this platform.

**Comment by skyejohnson6156** [comment]
I had a similar issue before with certain downloads triggering security warnings. It’s always best to double-check with multiple antivirus tools. Also, for my statistics assignments, I’ve been using [statistics assignment help](https://www.statisticsassignmenthelp.com/) from [www.statisticsassignmenthelp.com](https://www.statisticsassignmenthelp.com/), and they’ve been a lifesaver! Not only do they provide accurate solutions, but they also guide on safe data handling and software usage.
